src/js/toggleImportExport.js

```diff
@@ -0,0 +1,19 @@
+function importExportToggle() {
+	document.getElementById("Automationtemplates").click();
```

Comment:
Can you add a `?` after `getElementById("Automationtemplates")` to check for null value? So it should look like this:

`document.getElementById("Automationtemplates")?.click();`

On my environment I didn't have automation templates, so it didn't work (better check all of them)

Reply:
Ah, was running it under an Admin account (don't tell). Thanks for that, will get this sorted...

Reply:
> Hi @christh can you check my comment?
@ewerkman  Have made the code more defensive - please see updated PR, thanks!